R/computeacf.R

```diff
@@ -34,8 +34,8 @@ computeacf <- function(tseries, W.max, Lambda=100) {
 }
 
 ## generic function to plot an object of class "myGamma"
-plot.hadronacf <- function (Gamma, col = "black", ...)
-{
+plot.hadronacf <- function (x, col = "black", ...) {
```

Comment:
is this a requirement?

Reply:
I mean copying the argument.

Reply:
The requirement is that the argument is called `x`. I could have renamed it all through the function, but this way the chances of breaking something are smaller. We can rename it to `x` in the whole function independently of that.

Reply:
I see, I was just wondering since you've made replacements like this in several places.

Reply:
its required for the package for some reason...